packages/web/package.json

```diff
@@ -12,6 +12,8 @@
     "start:stage": "npm run write-sha && npm run publish-scripts && env-cmd ./.env/.env.git env-cmd --no-override ./.env/.env.stage turbo run start",
     "start:stage-local-trpc": "npm run write-sha && npm run publish-scripts && env-cmd ./.env/.env.git env-cmd --no-override ./.env/.env.stage-local-trpc turbo run start",
     "start:prod": "npm run write-sha && npm run publish-scripts && env-cmd ./.env/.env.git env-cmd --no-override ./.env/.env.prod env-cmd ./.env/.env.source-maps turbo run start",
+    "start:stage-local-trpc": "npm run write-sha && npm run publish-scripts && env-cmd ./.env/.env.git env-cmd --no-override ./.env/.env.stage-local-trpc turbo run start",
+    "start:prod-local-trpc": "npm run write-sha && npm run publish-scripts && env-cmd ./.env/.env.git env-cmd --no-override ./.env/.env.prod-local-trpc turbo run start",
```

Comment:
Looks like there are two `"start:stage-local-trpc"` entries.
One option instead of adding more `start` entries would be to do:

```
VITE_TRPC_ENDPOINT=http://localhost:2022/trpc npm run web:prod
```

of course configuring the trpc server to use remote postgres / elasticsearch is still manual... I added some notes in the trpc readme in [this PR](https://github.com/AudiusProject/audius-protocol/pull/7077/files#diff-e06ebb9005ef056f635a00b51a56661ca93b20491787ba850559bb8926de6314).

Anyway this is fine... just wondering if it's necessary to add more `start` variants.

Reply:
Yeah I think we'll decide on a final structure as a team, but I'll remove them for now in favor of the env variable, especially since you've added it to the README in your PR. Thanks!